README.md

```diff
@@ -329,6 +330,26 @@ descriptors:
           unit: second
 ```
 
+#### Example 5
+
+We can also define unlimited rate limit descriptors:
+
+```yaml
+domain: internal
+descriptors:
+  - key: ldap
+    rate_limit:
+      unit: second
```

Comment:
Is there a way we can remove the requirement to specify `unit`? It doesn't make sense in the unlimited context. Or perhaps: `unit: unlimited` if that is easier?

Reply:
Changed it so that specifying unit for unlimited descriptors is not needed